docs/src/tool/index.md

```diff
@@ -292,55 +321,56 @@ development API key), and sets `REACH_RPC_TLS_REJECT_UNVERIFIED` to
 Consider this example from the @{seclink("tut-7-rpc")} tutorial:
 
 ```cmd
-$ reach rpc-run python3 -u ./index.py
+$ ./reach rpc-run python3 -u ./index.py
 ```
 
 # {#ref-usage-docker-reset} `reach docker-reset`
 
-You can easily kill and remove all Docker containers by executing
+This command removes all containers and volumes from Docker. If you have any personal containers, those containers will also be removed. Containers can then be restarted in your Docker Images list. It does not uninstall any Docker images.
 
 ```cmd
-$ reach docker-reset
+$ ./reach docker-reset
 ```
 
 This can be a useful thing to try if your Docker containers stop responding to requests or otherwise misbehave, or if you have updated your Reach images (with `reach update`) but those changes are not taking effect.
-This command is a loose approximation of "turning Docker off and on again."
 It will affect all Docker containers on your machine, not just those created by Reach.
 
 # {#ref-usage-upgrade} `reach upgrade`
 
-You can upgrade your Reach installation by executing
+This command downloads the most recent `reachsh/reach-cli` Docker image only.
```

Comment:
That's technically not right - it updates the script as well:
https://github.com/reach-sh/reach-lang/blob/f8215673f9d66d5257575b2b392958da18a96c58/reach#L70

...but I think users needn't worry about those details and the original blurb was vague but okay.

(BTW I'm working on a replacement for `reach update` and `reach upgrade` that'll change this entirely.)

Reply:
In addition, I don't think we should refer to docker images by name in the documentation... they are internal implementation details. We should describe this generally as "upgrade your tool" (the tool being the thing this document is about)